src/tests/ftest/object/same_key_different_value.py

```diff
@@ -176,63 +159,57 @@ def test_array_to_single_value(self):
         for i in range(3):
             try:
                 # create an object and write array value data into it
-                obj = self.container.write_an_array_value(array_value_data,
-                                                          dkey, akey,
-                                                          obj_cls=1)
+                obj = self.container.write_an_array_value(array_value_data, dkey, akey, obj_cls=1)
                 # read the data back and make sure its correct
                 length = len(array_value_data[0])
                 read_back_data = self.container.read_an_array(
                     len(array_value_data), length + 1, dkey, akey, obj)
 
                 for j in range(3):
-                    if (array_value_data[j][0:length-1] !=
-                            read_back_data[j][0:length-1]):
+                    if (array_value_data[j][0:length - 1] != read_back_data[j][0:length - 1]):
                         self.log.info("Written Data: %s", array_value_data[j])
                         self.log.info("Read Data:    %s", read_back_data[j])
-                        self.fail("Data mismatch\n")
+                        self.fail("Data mismatch")
                 # test case 1
                 if i == 0:
                     try:
                         # write single value data to same keys, expected to fail
-                        self.container.write_an_obj(single_value_data,
-                                                    len(single_value_data)+1,
-                                                    dkey, akey, obj, obj_cls=1)
+                        self.container.write_an_obj(
```

Comment:
(pylint-no-member) Instance of 'TestContainer' has no 'write_an_obj' member
(pylint-no-member) Instance of 'list' has no 'write_an_obj' member